pywinauto/controls/common_controls.py

```diff
@@ -254,27 +254,37 @@ def rectangle(self, area="all"):
         remote_mem = RemoteMemoryBlock(self.listview_ctrl)
         rect = win32structures.RECT()
 
+        # If listview_ctrl has LVS_REPORT we can get access to subitems rectangles
+        is_table = self.listview_ctrl.has_style(win32defines.LVS_REPORT)
+
         if area.lower() == "all" or not area:
             rect.left = win32defines.LVIR_BOUNDS
         elif area.lower() == "icon":
             rect.left = win32defines.LVIR_ICON
         elif area.lower() == "text":
             rect.left = win32defines.LVIR_LABEL
         elif area.lower() == "select":
-            rect.left = win32defines.LVIR_SELECTBOUNDS
+            rect.left = win32defines.LVIR_BOUNDS if is_table else win32defines.LVIR_SELECTBOUNDS

```

Comment:
Not sure it really makes difference. If `LVIR_SELECTBOUNDS` works the same way, why changing it?

Reply:
LVM_GETSUBITEMRECT has not LVIR_SELECTBOUNDS flag, but for save functionality i add this line, so if item is cell it will return same what and "all" parameter of area

Reply:
OK, got it. Thanks!